README.md

```diff
@@ -291,8 +291,83 @@ result:
 
 ```
 
+## <a name="UsageAdditionalActions"></a> Usage - Additional Actions
+
+In addition to the actions above there are a few additional actions that we will discuss here.
```

Comment:
"In addition to the auto-generated actions described above, there are a handful of higher-level workflow actions that implement common functionality."